api/APIv2.md

```diff
@@ -56,6 +56,7 @@ In general, there is one directory for each API endpoint:
 * `logs`: The LogBundle message sent from Device to Controller containing internal device logs.
 * `apps/instanceid/{app-instance-uuid}/logs`: The LogBundle message sent from Device to Controller containing application console device logs.
 * `certs`: The ZControllerCert message is sent from Controller to Device, and contains the list of certificates used by Controller. Each ZControllerCert message replaces the current list on the device with the new list of certificates. Therefore, if an empty list is sent, it resets the list on the receiving side.
+* `uuid`: This API is used by device to fetch its unique idenitifier allocated by Controller. Along with uuid, reply for this request will also contain other device properties associated with the device, such as Manufacturer, Product Model, Enterprise, Device name etc.
```

Comment:
Some articles missing. "the controller", "the uuid", "the reply"

Also zedcloud does not currently send enterprise and device name in ConfigResponse, so we should probably remove those instead of propagating them here.

Reply:
> Should we also remove the ConfigRequest using a URL without the UUID? That is in the APIv4.md file

updated the document, pls check now.

Reply:
> Some articles missing. "the controller", "the uuid", "the reply"
> 
> Also zedcloud does not currently send enterprise and device name in ConfigResponse, so we should probably remove those instead of propagating them here.

updated the document, pls check now. Also removed enterprise and name fields from the proto definition